core_app/mls_integration.py

```
@@ -0,0 +1,314 @@
+    try:
+        if not os.path.exists(csv_path):
+            return {
+                'success': False,
+                'count': 0,
+                'message': f'MLS file not found: {csv_path}',
+                'last_updated': None
+            }
+        
+        # Load CSV data with error handling for malformed lines
+        df = pd.read_csv(csv_path, quotechar='"', skipinitialspace=True, on_bad_lines='skip')
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)